web/services/counter_service.py

```
@@ -0,0 +1,102 @@
+from django.db.models.signals import post_save
+from django.dispatch import receiver
+from django.utils import timezone
+
+from web.models import CounterStatistic, CourseProgress, Enrollment, UserActivity, WebRequest
+
+
+class CounterService:
+    """Service to manage counter statistics."""
+
+    @staticmethod
+    def update_active_users_count():
```

Ruff (0.8.2)

Missing return type annotation for staticmethod `update_active_users_count`

(ANN205)